### PR TITLE
Add troubleshooting section to readme

### PR DESCRIPTION
**Pull request by laurieboyes**
Readme addition that would have helped me

### CODE REVIEW

File: keycloak/docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     healthcheck:
       test: ['CMD', 'curl', '-f', 'http://localhost:8080/auth']
       start_period: 60s
-    image: jboss/keycloak:15.0.2
```
Comment (stephenkaye-ho) [inline keycloak/docker-compose.yml:8]: The version was originally set to match production. Can we ensure it is still the same as production

Comment (laurieboyes) [reply keycloak/docker-compose.yml:8]: Ah, it is almost certainly not 😐  I’ll revert this

**Review by stephenkaye-ho — APPROVED** [review]

**Review by taomoh — APPROVED** [review]
Tested this successfully on my macbook which has an Apple M2 chip

**Comment by laurieboyes** [comment]
Closing pull request

**Comment by laurieboyes** [comment]
Moved to https://github.com/UKHomeOffice/callisto-ui/pull/13

**Pull request closed without merging**